Atomic/atomic.py

```diff
@@ -516,6 +518,55 @@ def container_run_command(self):
         command += self.image
         return command
 
+    def ps(self):
```

Comment:
Can you split this out into a separate file ps.py, rather then continuously growing atomic.py.  I eventually would like to break each command into its own py file.